tests/libs/fftw/tests/rm_execution

```diff
@@ -11,17 +11,23 @@ fi
 check_rms
 rm=$RESOURCE_MANAGER
 
-NODES=2
-TASKS=8
-ARGS=8
+if [ -z "$GITHUB_ACTIONS" ]; then
```

Comment:
What about CirrusCI (for Rocky on aarch64) ?

Reply:
First goal was to get it working on GitHub Actions only. This can be easily changed later if we want to run it somewhere else.

That was my approach to this at least.

Reply:
According to https://cirrus-ci.org/guide/writing-tasks/#environment-variables you could use `CIRRUS_CI` to check whether it executes on Cirrus.